docs/howtos-howto.md

```diff
@@ -64,14 +50,16 @@ The workflow script for the Github action can be found at
 
 ## Resolving a conflict with an existing HOWTO
 
-> :warning: TODO: This is still in progress. Currently conflicts are resolved by
-            checking whether the Github action fails, and if so, manually making
-            the required fixes.
+TODO: This is still in progress. Currently conflicts are resolved by
+checking whether the Github action fails, and if so, manually making
+the required fixes.
+</div>
```

Comment:
Remove this?

Reply:
Oh? What fixed this? Do we have this test set up on Travis?

Reply:
No I mean removing the "</div>", this is markdown, right?

Reply:
Oh oops, yes! Done.